includes/wikia/services/PageStatsService.class.php

```diff
@@ -88,60 +59,40 @@ static function onArticleSaveComplete(
 	static function onArticleDeleteComplete(
 		WikiPage $article, User $user, $reason, $articleId
 	): bool {
-		wfProfileIn(__METHOD__);
-
-		$title = $article->getTitle();
-
-		// tell service to invalidate cached data for deleted page
-		if (!empty($title)) {
-			$service = self::newFromTitle($title);
-			$service->regenerateData();
-
-			wfDebug(__METHOD__ . ": cache cleared for page #{$articleId}\n");
-		}
+		$service = new self( $article->getTitle() );
```

Comment:
previously there was an emptiness check so we assumed it is possible that `$article->getTitle()` can return null. Not we don't do this check which means we can create new `PageStatsService` with empty title. But the `PageStatsService` does not check for emptiness when making calls like `$this->mTitle->getArticleID()`. Isn't this an issue?

Reply:
`WikiPage::doDeleteArticleReal` would fail if somehow the `WikiPage`  instance had a null title, so if we got here, it's safe to assume we have an instance of `Title`.